fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -135,6 +136,13 @@ public PlanFragment translatePlan(PhysicalPlan physicalPlan, PlanTranslatorConte
             rootFragment = exchangeToMergeFragment(rootFragment, context);
         }
         List<Expr> outputExprs = Lists.newArrayList();
+        if (physicalPlan instanceof PhysicalProject) {
+            PhysicalProject project = (PhysicalProject) physicalPlan;
+            if (isUnnecessaryProject(project)) {
+                List<Slot> slotReferences = removeAlias(project);
+                physicalPlan = (PhysicalPlan) physicalPlan.child(0).withOutput(slotReferences);
```

Comment:
no need to update child's output.  generate outputExprs according project's output's order is enough